doc/user-manual/getting-started/installation.rst

```diff
@@ -174,23 +174,34 @@ Nix or NixOS
 ------------
 
 Agda is part of the Nixpkgs collection that is used by
-https://nixos.org/nixos. To install Agda and agda-mode for Emacs,
-type:
+https://nixos.org/nixos. There are two ways to install Agda from nix:
 
-.. code-block:: bash
+* As Agda is a Haskell package available from hackage you can install
+  it like any other haskell package. To install Agda and agda-mode for
```

Comment:
I don't believe this will install agda-mode. I personally use the separate nix package for agda-mode with `emacsWithPackages`

Reply:
I tried `haskellPackages.agda` and `agdaWithPackages` and they both provided the `agda-mode` binary. But, I don't think it's worth mentioning really.

Reply:
I would expect that it's quite important because nearly noone uses agda without interactive editing, and emacs is one of the most popular editors. So even if it's just a short sentence saying that it will work without further ado, or with a simple change in the emacs config, that's already helpful. Noone will want to switch to a system where there is no editor support.

Reply:
Agreed. I find the support for developing programs interactively in emacs essential. I don't think you can get very far without it.

What I meant was that it is so essential that it's just part of agda and that it should be expected that it's part of the installation. I added a step to run the `agda-mode setup` after installing the package. (Analogously to the hackage install instructions above).